demo/index.html

```diff
@@ -19,53 +19,37 @@
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
-  <link rel="import" href="../../paper-styles/demo-pages.html">
+  <link rel="import" href="../../paper-styles/color.html">
+  <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+  <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
   <link rel="import" href="cats-only.html">
   <link rel="import" href="validatable-input.html">
-
-  <style is="custom-style">
-
-    .valid {
-      color: var(--google-green-500);
-    }
-
-    .invalid {
-      color: var(--google-red-500);
-    }
-
-  </style>
-
+  <custom-style>
+    <style is="custom-style" include="demo-pages-shared-styles"></style>
+  </custom-style>
 </head>
 <body>
-  <div class="vertical-section vertical-section-container centered">
-  <h1>&lt;iron-validatable-behavior&gt;</h1>
-
-  <template is="dom-bind">
-
-    <cats-only></cats-only>
-
-    <section>
-
-      <p>
-        only type <code>cats</code>:
-
-        <input is="validatable-input" invalid="{{invalid}}" validator="cats-only">
-
-        <span class="valid" hidden$="[[invalid]]">valid</span>
-        <span class="invalid" hidden$="[[!invalid]]">invalid</span>
-      </p>
-
-    </section>
-
-  </template>
-
+  <div class="vertical-section-container centered">
+    <h3>The <code>cats-only</code> validator only accepts the string "cats"</h3>
+    <demo-snippet class="centered-demo">
+      <template>
+        <custom-style>
+          <style is="custom-style" include="demo-pages-shared-styles">
+            validatable-input:not([invalid]) {
```

Comment:
I think this rule needs to come after `validatable-input[invalid]` because `:not()` doesn't increase specificity for some reason ([spec](https://www.w3.org/TR/css3-selectors/#specificity)).